source/puppet/4.10/lang_type_aliases.md

```diff
@@ -13,21 +12,21 @@ Type aliases allow you to create reusable and descriptive data and resource type
 Type aliases are written as:
 
 ``` puppet
-type <ALIAS NAME> = <TYPE DEFINITION>
+type <MODULE NAME>::<ALIAS NAME> = <TYPE DEFINITION>
 ```
 
-The `<ALIAS NAME>` begins with a capital letter and must not be a [reserved word][reserved].
+The `<MODULE NAME>` must be named after the module that contains the Type Alias, and both the `<MODULE NAME>` and `<ALIAS NAME>` begin with a capital letter and must not be a [reserved word][reserved].
```

Comment:
I think type alias should be lowercase here, looks like it is in other places in the doc.